ada_feeding_perception/config/table_detection.yaml

```diff
@@ -0,0 +1,8 @@
+# NOTE: You have to change this node name if you change the node name in the launchfile.
+table_detection:
+  ros__parameters:
+    # The rate at which to publish the pose of the detected table
+    rate_hz: 3.0
+
+    # A boolean to determine whether to visualize plate detection 
+    viz: True
```

Comment:
Nit: can you [add a newline to the end of this file](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)?